app/services/app_sign_up_service.rb

```diff
@@ -27,12 +27,14 @@ def create_user!
   end
 
   def create_access_token!
```

Comment:
In a v2 version of this endpoint, we should likely have this service return an Authorization Code and follow the authorization code grant flow. Likely through something like first-party apps (which we can't use at the moment because we allowed signup for all apps).